src/components/common/TreeTransfer.tsx

```diff
@@ -51,8 +51,6 @@ const TreeTransfer = ({ dataSource, targetKeys, ...restProps }: TreeTransferProp
               <Tree
                 blockNode
                 checkable
-                checkStrictly
-                defaultExpandAll
```

Comment:
Are you sure other component which also used this TreeTransfer
doesn't need those two props ?

Reply:
- checkStrictly cant't select all children when check thier parent
- defaultExpandAll I think if item's quantity is too many, select view will very long. I think fold it up would be fine